source/includes/_importing_content.md

```diff
@@ -103,7 +107,7 @@ This endpoint creates an item within a set. You can optionally specify a questio
 
 | Parameter | Description                                           |
 | --------- | ----------------------------------------------------- |
-| question  | Optional text representing the question being asked   |
+| question  | Optional text representing the question being asked.  |
```

Comment:
Period here, but not in `answer` description?

Reply:
eagle eye 🦅